File: PY/word_reverser.py
#Write a program that asks the user to input a sentence, 
# and then prints each word in reverse order, not reversed letters.

import os

def clear_screen():
    os.system("clear")

while True:

    clear_screen()

    print("Enter a sentence:")
    sentence = input()
    words = sentence.split()
    words.reverse()

    reversed_words = " ".join(words)

    print(f"\nYour reversed sentence is:\n{reversed_words}")

    choice = input("\nDo you want to try with another sentence? y/n:")

    if choice != "y":
        break

File: C#/wordReverser.cs
// Write a program that asks the user to input a sentence, 
// and then prints each word in reverse order, not reversed letters.

using System;

public class wordReverser
{
    public static void Run()
    {
        while (true)
        {
            Console.Clear();
            Console.Write("Write a sentence: ");
            string sentence = Console.ReadLine()!;

            List<string> words = new List<string>(sentence.Split(" ", StringSplitOptions.RemoveEmptyEntries));
            words.Reverse();

            Console.WriteLine("\nHere is your reversed sentence:");
            Console.WriteLine(string.Join(" ", words));

            Console.Write("\nDo you wish to play again? y/n :");

            string choice;
            choice = Console.ReadLine()!;

            if (choice != "y")
            {
                break;
            }
        }
    }
}